wire-gson-support/src/test/java/com/squareup/wire/GsonTest.java

```diff
@@ -98,7 +98,7 @@
 
   private static final String JSON_EXTENSIONS =
       ",\"squareup.protos.alltypes.ext_opt_int32\":2147483647,"
-          + "\"squareup.protos.alltypes.ext_opt_int64\":4611686018427388081,"
```

Comment:
I changed this test because previously we could use either hte adapter for int64 or uint64 and both would have been correct. With this different test value, the adapter for int64 returns a negative value, whereas uint64 is positive...